ethereum/contracts/ETHApp.sol

```diff
@@ -20,6 +21,12 @@ contract ETHApp is Application {
         totalETH = 0;
     }
 
+    function register(address _bridge) public override returns(bool) {
+        require(bridge == address(0), "Bridge has already been registered");
+        bridge = _bridge;
+        return true;
```

Comment:
Rather just make this a `void` method if it is going to return `true` under all expected conditions.

Reply:
as above, think its true in the case of ETHApp but may not be true for every app

Reply:
Went with the void + exception approach in https://github.com/Snowfork/polkadot-ethereum/pull/115/commits/ce891d0610f5a38410cd1ff327087dda266cc932